controllers/models/index.js

```diff
@@ -28,19 +28,19 @@ exports.retrieveTopics = () => {
     return retrieveData(sqlQuery)
 }
 
-exports.retrieveArticlesById = (id) => {
+exports.checkArticleExists = (id) => {
     const sqlQuery = `SELECT * FROM articles WHERE article_id=${id} ORDER BY created_at DESC`
     return retrieveData(sqlQuery)
 }
 
-exports.retrieveArticles = (query) => {
+exports.retrieveArticles = (query, includeBody) => {
     //COALESCE converts the value to 0 if its null
     //AS comment_count sets the column name
     //CAST(... AS INT) converts to an interger (apparently GROUP BY returns as a VARCHAR)
     //COUNT(*) counts all the items in the group
     //GROUP BY creates groups for all comments seperated by article_id
     //SQL hurts
-    let sqlQuery = format(`SELECT author, title, articles.article_id, topic, created_at, votes, article_img_url, COALESCE(comments.comment_count, 0) AS comment_count FROM articles
+    let sqlQuery = format(`SELECT author, title, articles.article_id,${includeBody ? ' body,':''} topic, created_at, votes, article_img_url, COALESCE(comments.comment_count, 0) AS comment_count FROM articles
```

Comment:
this is your project so it's up to you if you want to add in functionality like this, but I'm not sure any tickets ask for the ability to include the body in all articles?